package.json

```diff
@@ -62,6 +61,7 @@
     "rollup": "^3.20.2",
     "rollup-plugin-dynamic-import-variables": "^1.1.0",
     "typedoc": "^0.23.28",
-    "typescript": "^5.0.3"
+    "typescript": "^5.0.3",
+    "minimist": "^1.2.8"
```

Comment:
I was going to use the built-in arg parser in node, but it's marked as 'experimental' - so adding this single-file dependency in the mean time